velox/vector/SimpleVector.h

```diff
@@ -181,6 +182,8 @@ class SimpleVector : public BaseVector {
     } else {
       if constexpr (std::is_same<T, std::shared_ptr<void>>::value) {
         out << "<opaque>";
+      } else if (isDecimalKind(this->typeKind())) {
+        out << decimalToString<T>(valueAt(index), type());
```

Comment:
I see now where this function is used. It would be nice to add a small test to velox/vector/tests/VectorToStringTest.cpp . It seems this would belong to the PR that introduced decimalToString method.

Reply:
@mbasmanova We will add another PR to make these Decimal Vectors and add a test to `VectorToStringTest.cpp`

Reply:
@majetideepak will address in a separate PR.